### DISCUSSION TITLE
在数莓派用boringssl编译make -j出错

### DISCUSSION THREAD

**Discussion by mingyuan-ustc** [alibaba/xquic#225] [Q&A] [answered]
/home/pi/xquic-main/src/transport/xqc_conn.h:161:47: error: left shift count >= width of type [-Werror=shift-count-overflow]
  161 |     XQC_CONN_FLAG_VALIDATE_REBINDING    = 1UL << XQC_CONN_FLAG_VALIDATE_REBINDING_SHIFT,
      |                                               ^~
In file included from /home/pi/xquic-main/src/transport/xqc_packet_in.c:7:
/home/pi/xquic-main/src/transport/xqc_conn.h:161:47: error: left shift count >= width of type [-Werror=shift-count-overflow]
  161 |     XQC_CONN_FLAG_VALIDATE_REBINDING    = 1UL << XQC_CONN_FLAG_VALIDATE_REBINDING_SHIFT,
      |                                               ^~
/home/pi/xquic-main/src/transport/xqc_quic_lb.c:66:1: error: unknown type name ‘__uint128_t’; did you mean ‘__uint32_t’?
   66 | __uint128_t
      | ^~~~~~~~~~~
      | __uint32_t
/home/pi/xquic-main/src/transport/xqc_quic_lb.c:67:17: error: unknown type name ‘__uint128_t’; did you mean ‘__uint32_t’?
   67 | xqc_expand_left(__uint128_t left, __uint128_t right)
      |                 ^~~~~~~~~~~
      |                 __uint32_t

**Comment by ruiqizhou** [comment, marked as answer]
v1.2.0-stable 已修复，拉下最新的代码试试
